### ISSUE TITLE
Error with Style/WordArray cop

### ISSUE THREAD

**Issue by Ana06** [rubocop-hq/rubocop#4141]
Rubocop fails due to WordArrayCop. It seems to fail for all the offences of this cop.

## Failure

```
An error occurred while Style/WordArray cop was inspecting /home/Ana/Documents/software-o-o/app/controllers/package_controller.rb:88:57.
An error occurred while Style/WordArray cop was inspecting /home/Ana/Documents/software-o-o/app/controllers/package_controller.rb:117:74.
An error occurred while Style/WordArray cop was inspecting /home/Ana/Documents/software-o-o/app/controllers/package_controller.rb:120:56.
An error occurred while Style/WordArray cop was inspecting /home/Ana/Documents/software-o-o/app/controllers/package_controller.rb:121:66.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.47.1 (using Parser 2.4.0.0, running on ruby 2.2.5 x86_64-linux-gnu)

```

### Failed lines

 home/Ana/Documents/software-o-o/app/controllers/package_controller.rb:88
``` Ruby
    @related_categories = @related_categories.reject{|c| ["GNOME", "KDE", "Qt", "GTK"].include? c[:name] }
```

home/Ana/Documents/software-o-o/app/controllers/package_controller.rb:117
``` Ruby
      {:name => "Education & Science", :id => "Education", :categories => ["Education", "Science"]},
```

home/Ana/Documents/software-o-o/app/controllers/package_controller.rb:120
``` Ruby
      {:name => "Tools", :id => "Tools", :categories => [ "Network", "Settings", "System", "Utility"]},
```

home/Ana/Documents/software-o-o/app/controllers/package_controller.rb:121
``` Ruby
      {:name => "Multimedia", :id => "Multimedia", :categories => ["AudioVideo", "Audio", "Video", "Graphics"]},
```

## Steps to reproduce the problem

Run `rubocop` after having run `rubocop --auto-gen-config` to add Rubocop to a project.

## RuboCop version

```
0.47.1 (using Parser 2.4.0.0, running on ruby 2.2.5 x86_64-linux-gnu)
```

**Comment by Drenmi** [comment]
Sometimes when using `--auto-gen-config`, this cop gets a `MinSize` configuration of `-Infinity`, which makes it break. I vaguely recall there being an issue opened about this. Will see if I can find it tomorrow. 🙂

**Comment by backus** [comment]
@Drenmi I thought that was fixed? See https://github.com/bbatsov/rubocop/issues/2979 and https://github.com/bbatsov/rubocop/issues/2740#issuecomment-275007739

**Comment by Drenmi** [comment]
@backus: Happened to me again yesterday. 🙁

**Comment by jonas054** [comment]
@Ana06 @Drenmi Is this problem reproducible on the latest version (0.49.1)?

**Comment by Drenmi** [comment]
@jonas054: This happens intermittently for me. Still haven't been able to reliably reproduce it. 😕 

I am on the side of Alex here. I don't think the feature is useful. (Basing the `todo` on length of the array.) It also seems to intermittently break. The reward for the amount of complexity added is not nearly worth it, IMHO. 🙂

**Comment by bbatsov** [comment]
I'm closing this ticket due to no recent activity. Feel free to re-open it if you ever come back to it.
